### PR TITLE
Build by travis

### PR DESCRIPTION
**Pull request by michz**
Build docker images by Travis CI and push them to Docker Hub.

Matrix build with several versions.

Ansible is installed/build via pip as Alpine has only one version in its package manager per Alpine release.

### CODE REVIEW

**Review by twojtylak — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.1.84 ANSIBLE_VERSION=2.5.5
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.3.0.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.4.1.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.5.5
+
+script:
+  - docker build
+    --build-arg PACKER_VERSION=$PACKER_VERSION
+    --build-arg ANSIBLE_VERSION=$ANSIBLE_VERSION
+    -t solutiondrive/docker-aws-packer-ansible:packer$PACKER_VERSION-ansible$ANSIBLE_VERSION-awscli$AWSCLI_VERSION-inspec$INSPEC_VERSION .
```
Comment (sd-lueckel) [inline .travis.yml]: i would like it to tag the highest possible version from each application at latest (PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.2.16 ANSIBLE_VERSION=2.5.5 should be latest at the moment)

**Review by sd-lueckel — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (michz) [reply .travis.yml]: Like it. Already thought about this.

Comment (michz) [reply .travis.yml]: Do you think we should build some versions less? The matrix seems a bit to large for me, at least larger than really needed.

Comment (sd-lueckel) [reply .travis.yml]: I think we do not need to support ansible 2.3 and also inpsec 0.9

Comment (michz) [reply .travis.yml]: Are you sure with ansible? Currently/lately we *only* support 2.3 in this container. To preserve fully compatibility with current infrastructure using this container I'd suggest to keep ansible 2.3.
With inspec I agree, I just was not sure if we really do not use 0.9.

**Comment by michz** [comment]
Tagging `latest` works now as expected by @sd-lueckel .

File: .travis.yml (hunk 1 above)
Comment (sd-lueckel) [reply .travis.yml]: Yeah, ok! But would drop it as soon as possible

**Review by sd-lueckel — APPROVED** [review]

**Review by mattagohni — APPROVED** [review]

**Pull request merged**